compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -410,8 +417,14 @@ int entry(int argc, char **argv)
   {
     auto graph = module->graph(idx);
 
-    // quantize the graph
-    quantizer.quantize(graph);
+    if (arser["--warn_weights"])
+    {
+      quantizer.check(graph);
+    }
+    else
+    { // quantize the graph
+      quantizer.quantize(graph);
+    }
```

Comment:
when `warn_weights` is used, only check is done?
I'd prefer not use use `else`

Reply:
or if your intension is to just "check" then it would be better to describe it in the help message

Reply:
My intention with this version is to run the weights check only once.

Reply:
Could you describe more about what `only once` means?
I don't understand why do `quantizer.check` and then do save it to a file.

Reply:
I only want to check constants, not activations